### ISSUE TITLE
Ryzen 3000 issues

### ISSUE THREAD

**Issue by pgsill** [ArcadeRenegade/SidebarDiagnostics#236]
**1. Your Operating System**
Win 10
**2. App Version**
Latest
**3. Any Relevant Settings**
No
**4. Steps to Reproduce**
Running a Ryzen 3700X, I experience the following issues:

- No temperature readings
- No voltage readings
- No fan speed
- No clock readings
- High voltage use due to frequent polling (see https://www.reddit.com/r/Amd/comments/cbls9g/the_final_word_on_idle_voltages_for_3rd_gen_ryzen/)

Basically, the only thing that does work is load %. Voltages are higher than usual when nothing but SD is running. Closing SD drops the voltage down to idle levels.

**Comment by ArcadeRenegade** [comment]
Please see #187 for the voltage thing maybe increasing the time between polls in the settings will help.

**Comment by pgsill** [comment]
I'll keep an eye on that issue, thanks!
